system/mrm_comfortable_stop_operator/src/mrm_comfortable_stop_operator/mrm_comfortable_stop_operator_core.cpp

```diff
@@ -65,6 +71,21 @@ void MrmComfortableStopOperator::operateComfortableStop(
   }
 }
 
+rcl_interfaces::msg::SetParametersResult MrmComfortableStopOperator::onParameter(
+  const std::vector<rclcpp::Parameter> & parameters)
+{
+  using autoware::universe_utils::updateParam;
+  updateParam<double>(parameters, "min_acceleration", params_.min_acceleration);
+  params_.min_acceleration = std::abs(params_.min_acceleration);
```

Comment:
Why do you use `std::abs` here? :thinking: 
It looks like [the original code](https://github.com/autowarefoundation/autoware.universe/pull/9353/files#diff-8cbb3f9e143a55d21060bfcc6cbd6bc985262024e022991f0ac39fb688187d17R27) doesn't do that.

Reply:
@TomohitoAndo my bad, I thought the original always changed it to abs value, but it was a mistake when looking at a different module. I have removed that line!